docs/organizations/using-coding-standards.md

```diff
@@ -1,41 +1,43 @@
+Coding standards store your preferred tool and code pattern configurations and let you apply them to multiple repositories, ensuring consistent code quality and security across your organization.
 
-Create coding standards on your organization to define different tool and code pattern configurations that apply to a set of programming languages. After creating a coding standard, apply it to your repositories:
+This page covers the following topics:
 
--   Each repository can only follow one coding standard at a time. Applying a new coding standard to a repository <span class="skip-vale">unassigns</span> any previously applied coding standard.
--   Applying a coding standard to a repository only affects the configurations of the tools that are included in the coding standard, while the remaining tool and code pattern configurations of the repository remain unchanged.
--   If you later edit the tool and code pattern configurations in a coding standard, the repositories following that coding standard automatically start using the updated configurations on the next analysis.
--   When you customize the tools or code patterns of a repository that follows a coding standard, Codacy warns you that the repository will stop following the coding standard and asks for your confirmation.
-
-Optionally, set a default coding standard to apply it automatically to all new repositories that you add to Codacy. If no coding standard is set as default, new repositories don't follow any coding standard and use the Codacy default configurations instead.
-
-!!! important
-    Coding standards turn tools with configuration files on and off. Those tool configuration files, however, take precedence over the code patterns defined on the coding standard.
+-   [Creating a coding standard](#creating)
+-   [Editing a coding standard](#editing)
+-   [Deleting a coding standard](#deleting)
+-   [Defining a default coding standard](#set-default)
+-   [Using a coding standard alongside tool configuration files](#using-with-tool-configuration)
 
 ## Creating a coding standard {: id="creating"}
 
+!!! info "Codacy currently supports up to 10 coding standards per organization"
+
 To create a coding standard for your organization:
 
 1.  Open your organization **Policies** page, tab **Coding standards**.
 
-1.  Click the button **Create new standard** at the top right-hand corner of the page. This opens a window with the coding standard creation form.
-
-    !!! note
-        Codacy currently supports up to 10 coding standards per organization.
+1.  Click the button **Create new standard** at the top right-hand corner of the page. This opens the coding standard creation form.
 
 1.  Enter a unique name and click **Create standard**.
 
-    Optionally, select a repository for Codacy to use as a baseline when bootstrapping the tool and pattern configurations for the new coding standard. This is useful if you already have a configured repository that you wish to use as a template.
```

Comment:
I think this comment is still very helpful and should be kept: "This is useful if you already have a well-configured repository that you wish to use as a template"